Envelopes/graysonsolis_Toggle ReaPitch shift (cents) envelope.lua

```diff
@@ -0,0 +1,96 @@
+-- @description Toggle ReaPitch shift (cents) envelope
+-- @author Grayson Solis
+-- @version 1.0
+-- @about
+--   Toggles Shift (cents) envelope lane for ReaPitch on selected track. Adds ReaPitch if missing. Similar to the "volume" or "pan" default track envelopes.
+--   The parameter used can be toggled by editing the lines below!
+
+reaper.PreventUIRefresh(1)
+reaper.Undo_BeginBlock()
+
+local track = reaper.GetSelectedTrack(0, 0)
+if not track then
+  reaper.PreventUIRefresh(0)
```

Comment:
By the way, while this line here (and same a bit below) is correctly undoing the `PreventUIRefresh` from above, it forgets to the the same with `Undo_BeginBlock`.

An easy solution would be to wait until later before enabling `PreventUIRefresh` and `Undo_BeginBlock`. :wink: 

Thankfully REAPER has a fail-safe against scripts doing this kind of mistake so there is no ill effects.

Reply:
My bad, nope it is not.